LayoutTests/imported/w3c/web-platform-tests/notifications/constructor-basic.https.html

```diff
@@ -18,4 +18,22 @@
         notification.close()
     }
 }, "Called the notification constructor with one argument.")
+
+test(() => {
+    assert_equals(
+        new Notification("a").silent,
+        null,
+        "Expected null by default"
+    );
+}, "Constructing a notification without a NotificationOptions defaults to null");
+
+test(() => {
+    for (const expected of [null, true, false]) {
```

Comment:
It would be good to also try undefined here and maybe some others such as a number of a string.

Reply:
Good idea. Added.